Installations.md

```diff
@@ -86,13 +90,13 @@ If you need to update rails, you can run `bundle update rails`.  If you run into
 
 ### Step 3: Set up the database
 
-* We using the database used in WebsiteOne. Ask for a copy of `config/database.yml`the [websiteone slack channel](https://agileventures.slack.com/archives/C029E8G80).
+rails db:create
```

Comment:
Hi @ethanstrominger, big thank you for updating these database set up instructions in the readme. We have however created an #70 to have this better clarified. Maybe we can update that from there?